### ISSUE TITLE
bug(E1101): simple pyfancy usage

### ISSUE THREAD

**Issue by Kristinita** [PyCQA/pylint#2448]
### 1. Summary

Pylint check, that simply [**Pyfancy**](https://github.com/ilovecode1/Pyfancy-2#usage) usage is wrong.

### 2. Data

+ [**Online demonstration on Repl.it**](https://repl.it/@Kristinita/sashapylintpyfancye1101py).

+ `sasha__pylint__pyfancy_e1101.py`:

```python
"""Docstring."""
from pyfancy.pyfancy import pyfancy

print(pyfancy().yellow().bold('Kira Goddess!'))

```

+ output:

```text
Kira Goddess!
```

Output yellow and bold:

![Pyfancy Kira Goddess](https://i.imgur.com/jkiry4N.png)

### 3. Steps to reproduce

```text
pylint sasha__pylint__pyfancy_e1101.py
```

### 4. Expected behavior

No warnings and/or errors.

### 5. Actual behavior

```text
D:\SashaDebugging\SashaPython>pylint sasha__pylint__pyfancy_e1101.py
************* Module sasha__pylint__pyfancy_e1101
sasha__pylint__pyfancy_e1101.py:4:6: E1101: Instance of 'pyfancy' has no 'green' member (no-member)

----------------------------------------------------------------------
Your code has been rated at -15.00/10 (previous run: -15.00/10, +0.00)
```

### 6. Environment

```text
D:\SashaDebugging\SashaPython>pylint --version
pylint 2.1.1
astroid 2.0.4
Python 3.7.0 (v3.7.0:1bf9cc5093, Jun 27 2018, 04:59:51) [MSC v.1914 64 bit (AMD64)]
```

Issue doesn't fix, if `pip install pylint astroid --pre -U`.

Thanks.
